File: pgm/ABC_Visual_Basic/ABC_168_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim N As Integer
        N = Console.ReadLine()
        If {2, 4, 5, 7, 9}.Contains(N Mod 10) Then
            Console.WriteLine("hon")
            Return
        ElseIf {0, 1, 6, 8}.Contains(N Mod 10) Then
            Console.WriteLine("pon")
            Return
        Else
            Console.WriteLine("bon")
            Return
        End If
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_168_A.cs
using System;
using System.Linq;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int N;
        N = int.Parse(arr[0]);
        if (new int[] { 2, 4, 5, 7, 9 }.Contains(N % 10))
            Console.WriteLine("hon");
        else if (new int[] { 0, 1, 4, 6, 8 }.Contains(N % 10))
            Console.WriteLine("pon");
        else
            Console.WriteLine("bon");
        return;
    }
}